src/game/client/components/hud.cpp

```diff
@@ -628,11 +628,29 @@ void CHud::RenderCursor()
 
 	vec2 Pos = *m_pClient->m_pCamera->GetCenter();
 	RenderTools()->MapScreenToGroup(Pos.x, Pos.y, Layers()->GameGroup(), m_pClient->m_pCamera->GetZoom());
-	Graphics()->TextureSet(g_pData->m_aImages[IMAGE_GAME].m_Id);
+	IGraphics::CTextureHandle *Texture = &g_pData->m_aImages[IMAGE_GAME].m_Id;
```

Comment:
pTexture

Reply:
why should I grap a copy? Oh it's a handle 👍

Reply:
👍